File: 241022/알파벳에 따른 평가/evaluation-by-alphabet.cpp
#include <iostream>
using namespace std;

int main() {
    char letter;
    cin >> letter;

    if (letter == 'S') {
        cout << "Superior";
    } else if (letter == 'A') {
        cout << "Excellent";
    } else if (letter == 'B') {
        cout << "Good";
    } else if (letter == 'C') {
        cout << "Usually";
    } else if (letter == 'D') {
        cout << "Effort";
    } else {
        cout << "Failure";
    }
    return 0;
}

File: 241022/알파벳에 따른 평가/evaluation-by-alphabet.js
const fs = require("fs");
let letter = fs.readFileSync(0).toString().trim();

if (letter === "S") {
    console.log("Superior");
} else if (letter === "A") {
    console.log("Excellent");
} else if (letter === "B") {
    console.log("Good");
} else if (letter === "C") {
    console.log("Usually");
} else if (letter === "D") {
    console.log("Effort");
} else {
    console.log("Failure");
}